l10n_it_fatturapa_in_rc/tests/test_fatturapa_in_rc.py

```diff
@@ -118,31 +114,20 @@ def test_00_xml_import(self):
         self.assertEqual(invoice.invoice_line_ids[1].name, "BANCALI")
         self.assertFalse(invoice.invoice_line_ids[0].rc)
         self.assertTrue(invoice.invoice_line_ids[1].rc)
+        self.assertEqual(invoice.invoice_line_ids[0].tax_ids.name, "22% e-bill")
         self.assertEqual(
-            invoice.invoice_line_ids[0].invoice_line_tax_ids.name, "22% e-bill"
-        )
-        self.assertEqual(
-            invoice.invoice_line_ids[1].invoice_line_tax_ids.name,
+            invoice.invoice_line_ids[1].tax_ids.name,
             "Tax 22% Purchase RC ITA",
         )
         self.assertEqual(invoice.amount_total, 30.5)
         self.assertEqual(invoice.get_tax_amount_added_for_rc(), 4.4)
         self.assertEqual(invoice.amount_tax, 5.5)
         self.assertEqual(invoice.e_invoice_amount_tax, 1.1)
-        invoice.action_invoice_open()
+        invoice.action_post()
         self.assertAlmostEqual(invoice.rc_self_invoice_id.amount_total, 24.4)
         self.assertEqual(invoice.rc_self_invoice_id.amount_tax, 4.4)
         self.assertEqual(invoice.rc_self_invoice_id.amount_untaxed, 20)
         self.assertEqual(
-            invoice.rc_self_invoice_id.invoice_line_ids.invoice_line_tax_ids.name,
+            invoice.rc_self_invoice_id.invoice_line_ids.tax_ids.name,
             "Tax 22% Sales RC ITA",
         )
-
```

Comment:
Puoi restorare questa parte di test?
È stata espressamente aggiunta in https://github.com/OCA/l10n-italy/pull/1502 per risolvere un problema

Reply:
Ok.

Reply:
👍🏻